pkg/fleets/controller.go

```diff
@@ -531,6 +531,17 @@ func (c *Controller) updateFleetStatus(fleet *agonesv1.Fleet) error {
 		fCopy.Status.ReservedReplicas += gsSet.Status.ReservedReplicas
 		fCopy.Status.AllocatedReplicas += gsSet.Status.AllocatedReplicas
 	}
+	if runtime.FeatureEnabled(runtime.FeaturePlayerTracking) {
+		// to make this code simpler, we will loop around the gsSet list twice.
```

Comment:
simpler while the feature gate is in place? Do you want to add a TODO to consolidate this with the above loop when the feature graduates to ga?

Reply:
> simpler while the feature gate is in place? 

Yeah exactly. Let me be explicit about this. Excellent idea on the TODO, doing that now.